rasa/cli/x.py

```diff
@@ -38,7 +38,7 @@ def add_subparser(
 
     if is_rasa_x_installed():
         # we'll only show the help msg for the command if Rasa X is actually installed
-        x_parser_args["help"] = "Start Rasa X and the Interface"
+        x_parser_args["help"] = "Starts Rasa X and the interface."
```

Comment:
Is rasa x more than the interface (at least that users would be concerned with)? maybe "starts the rasa x interface"